src/components/fileUploader/index.tsx

```
@@ -82,19 +83,43 @@ const FileUploader: React.FC<FileUploaderProps> = ({
+            <Preview
+              // className={files.length == 1 ? "md:col-start-2" : ""}
+              key={index}
+              file={file}
+            />
           ))}
 
-          {filesHolder.length > 0 ? (
-            <div id="no-drag" className="m-auto h-44 content-center">
+          {filesHolder.length > 0 && props.maxFiles ? (
+            props.maxFiles > 1
+          ) : 1 ? (
```

Biome (1.9.4)

[error] Unexpected constant condition.



(lint/correctness/noConstantCondition)